MiniApp/Classes/Display/RealMiniAppView+WKUIDelegate.swift

```diff
@@ -0,0 +1,48 @@
+import WebKit
+
+extension RealMiniAppView: WKUIDelegate {
+    func webView(_ webView: WKWebView,
+                 runJavaScriptAlertPanelWithMessage message: String,
+                 initiatedByFrame frame: WKFrameInfo,
+                 completionHandler: @escaping () -> Void) {
+        let alertController = UIAlertController(title: "Info", message: message, preferredStyle: .alert)
```

Comment:
I see in Android it displays something like this, 
Title - **The page at "<URL>"**
Message - **MESSAGE**

Yeah, we can do similar behavior in iOS. But it would expose the URL loaded in the WebView, is that ok?

Reply:
Now I'm thinking that we should change Android to use a custom Alert dialog too. We shouldn't be exposing the page URL. I can't think of anything that makes sense to use as the title, except maybe the title of the Mini App. Is there any way to access that from here? It seems that it might make the code a lot more complicated just to inject the Mini App title...

I also think setting the title as just an empty string might be okay. Maybe we should confirm with the product side.

Reply:
Isn't it possible to receive title as an input?

Reply:
@ts-nitish-a-jain JS dialog doesn't have option to have a title

Reply:
how about making the title to be the display name of the mini app?

Reply:
Cory sent a message to check on the product side if that is ok.

Reply:
Added Mini app displayname as JS dialog title